.github/workflows/tests.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       name: Install dependencies
       run: |
         # Install mavlink-router
```

Comment:
This line is not from the PR itself, but it would be good to remove or modify it, because the CI is interpreting all commands below it (apt, npm, etc) as if they were under it.

Reply:
Done.